src/components/Admin/ManageSensors/DeleteSensorModal.tsx

```diff
@@ -239,7 +241,7 @@ const DeleteSensorModal: ({sensors}: DeleteSensorModalProps) => JSX.Element = ({
               <Box>
                 <LabelValue
                   label={t('sensors.purpleAirId')}
-                  value={purpleAirId}
+                  value={purpleAirId.toString()}
```

Comment:
Should this be replaced with `numberToString()` from Util/Util.ts in case `Number.NaN` every sneaks in?

Reply:
Yes good catch